OneSignalExample/Assets/OneSignal/README.md

```diff
@@ -1,35 +1,112 @@
 
-#### Unity Versions
-* Unity 2018.4, 2019.4, and 2020.2
+#### Table of Contents
+- [Requirements](#requirements)
+- [Push Notification Credentials](#push-notification-credentials)
+- [Installation](#installation)
+  - [Unity Asset Store](#installation)
+  - [Unity Package Manager](#installation)
+- [Platform Configuration](#platform-configuration)
+  - [iOS](https://documentation.onesignal.com/docs/unity-sdk-setup#step-5---ios-setup)
+  - [Android](https://documentation.onesignal.com/docs/unity-sdk-setup#step-6---android-setup)
+- [Usage](#usage)
+  - [Initialization](#initialization)
 
-### Quick setup
-Add the following code to a script that starts with your game or app.
-```C#
-  // Replace 'YOUR_ONESIGNAL_APP_ID' with your OneSignal App ID from app.onesignal.com
-  OneSignal.StartInit("YOUR_ONESIGNAL_APP_ID").EndInit();
-```
+## Requirements
+- A [OneSignal Account](https://app.onesignal.com/signup) if you do not already have one
+- Your OneSignal App ID which you can find under **Settings > Keys & IDs**
+- Unity 2018.4 or newer
+- In order to test push notifications you will need
+  - An Android 4.0.3 or newer device or emulator with "Google Play services" installed
+  - An iOS 9 or newer device (iPhone, iPad, or iPod Touch)
+
+## Push Notification Credentials
+You must generate the appropriate credentials for the platform(s) you are releasing on:
+
+- iOS - [Generate an iOS Push Certificate](https://documentation.onesignal.com/docs/generate-an-ios-push-certificate)
+- Android - [Generate a Google Firebase Server API Key](https://documentation.onesignal.com/docs/generate-a-google-server-api-key)
+- Amazon Fire - [Generate an Amazon API Key](https://documentation.onesignal.com/docs/generate-an-amazon-api-key)
+
+## Installation
+There are two methods of installation available for the OneSignal Unity SDK:
+<details>
+    <summary>Unity Asset Store</summary>
+
+> *This installation method is recommended if you are upgrading from any version 2.13.4 or older. See more on this in Step 5.*
+
+1. Add the OneSignal Unity SDK as an available asset to your account by clicking **Add to My Assets** from [our listing on the Unity Asset Store](https://assetstore.unity.com/packages/add-ons/services/billing/onesignal-sdk-193316).
+2. Find the package waiting for you to download by clicking **Open in Unity** from that same page. This will open the Unity Editor and its Package Manager window.
+3. On the SDK's listing in the Editor click the **Download** button. When it finishes click **Import**.
+
+    ![onesignal unity sdk in my assets](Documentation~/asset_listing.png)
+
+4. A prompt to import all of the files of the OneSignal Unity SDK will appear. Click **Import** to continue and compile the scripts into your project.
+5. Navigate to **Window > OneSignal** (or follow the popup if upgrading) in the Unity Editor which will bring up a window with some final steps which need 
```

Comment:
Does Windows have the same menu layout? I know macOS has a Window top-level menu, but I'm unsure if Windows has this.

Reply:
It's the same on both platforms for Unity Editor.